packages/eslint-plugin/src/rules/call-super-on-override.ts

```diff
@@ -0,0 +1,119 @@
+import type { TSESTree } from '@typescript-eslint/utils';
+import { AST_NODE_TYPES } from '@typescript-eslint/utils';
+
+import * as utils from '../util';
+
+type MessageIds = 'missingSuperMethodCall';
+
+export default utils.createRule<[], MessageIds>({
+  name: 'call-super-on-override',
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description:
+        'Require overridden methods to call super.method in their body',
+      recommended: false,
+      requiresTypeChecking: false,
+    },
+    messages: {
+      missingSuperMethodCall:
+        "Use 'super{{property}}{{parameterTuple}}' to avoid missing super class method implementations",
+    },
+    fixable: 'code',
+    schema: [
+      {
+        type: 'object',
+        properties: {
+          topLevel: {
+            type: 'boolean',
+          },
+        },
+        additionalProperties: false,
+      },
+    ],
+  },
+  defaultOptions: [],
+  create(context) {
+    return {
+      'MethodDefinition[override=true][kind="method"]'(
+        node: TSESTree.MethodDefinition,
+      ): void {
+        let methodName = '',
+          methodNameIsLiteral = false,
+          methodNameIsNull = false; // don't add quotes for error message on [null] case
+
+        if (node.key.type === AST_NODE_TYPES.Identifier) {
+          methodName = node.key.name;
+        } else {
+          methodNameIsLiteral = true;
+          // null & undefined can be used as property names, undefined counted as Identifier & null as Literal
+          methodName =
+            (node.key as TSESTree.Literal).value?.toString() ?? 'null';
+          methodNameIsNull = (node.key as TSESTree.Literal).value == null;
+        }
+
+        const { computed: isComputed } = node,
+          bodyStatements = node.value.body!.body;
+
+        // Search for super method call
+        for (const statement of bodyStatements) {
+          if (
+            isSuperMethodCall(
+              statement,
+              methodName,
+              !methodNameIsLiteral && isComputed,
+            )
+          ) {
+            return; // We are done here, no missingSuperMethodCall error
```

Comment:
[Nit] This is self-evident from the code, I think.

```suggestion
            return;
```